docs/source/whatsnew_1_5.md

```diff
@@ -0,0 +1,56 @@
+
+# What's new in 1.5 🎉🎉
+
+- MAISI inference accelerate
+- Support numpy 2.x and Pytorch 2.6
```

Comment:
Perhaps we need switch the order of the two highlights to keep it to be consistent with the content below.